core/test/scripts/Voting.js

```diff
@@ -43,6 +78,12 @@ contract("scripts/Voting.js", function(accounts) {
     // Give voter all of the tokens.
     const initialSupply = await votingToken.balanceOf(account1);
     await votingToken.transfer(voter, initialSupply);
+
+    // Add supported identifiers
+    for (k = 0; k < Object.keys(TEST_IDENTIFIERS).length; k++) {
```

Comment:
two nits: 
- `let k = 0`
- Can this be a `for ... in` loop instead? Or is there a reason that wouldn't work for this case?

Also just a random question, why `k` rather than `i`? (No need to change, just genuinely curious :) ).

Reply:
I used `k` initially to represent a "key" but I changed in favor of `i` for consistency and to better represent it as just an iterator